secrets/dataplane-observability-config.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Let's include a .sample file for the configuration documentation. But let's first discuss the configuration design